discord/voicetwine/egg-pterodactyl-voicetwine.json

```diff
@@ -0,0 +1,152 @@
+      "container": "node:bookworm-slim",
+      "entrypoint": "bash"
+    }
+  },
+  "variables": [
+    {
+      "name": "Node Environment",
+      "description": "The environment setting to use for Node.js",
+      "env_variable": "NODE_ENV",
+      "default_value": "production",
+      "user_viewable": false,
+      "user_editable": false,
+      "rules": "required|string|in:development,production",
+      "field_type": "text"
+    },
+    {
+      "name": "Log Level",
+      "description": "Logging level for VoiceTwine",
+      "env_variable": "LOG_LEVEL",
+      "default_value": "info",
+      "user_viewable": true,
+      "user_editable": true,
+      "rules": "required|string|in:debug,info,warn,error",
+      "field_type": "text"
+    },
+    {
+      "name": "MariaDB Host",
+      "description": "MariaDB server host address",
+      "env_variable": "MARIADB_HOST",
+      "default_value": "172.18.0.1",
+      "user_viewable": true,
+      "user_editable": true,
+      "rules": "required|string",
+      "field_type": "text"
+    },
+    {
+      "name": "MariaDB Port",
+      "description": "MariaDB server port",
+      "env_variable": "MARIADB_PORT",
+      "default_value": "3306",
+      "user_viewable": true,
+      "user_editable": true,
+      "rules": "required|numeric",
+      "field_type": "text"
+    },
+    {
+      "name": "MariaDB User",
+      "description": "MariaDB username",
+      "env_variable": "MARIADB_USER",
+      "default_value": "voicetwine",
+      "user_viewable": true,
+      "user_editable": true,
+      "rules": "required|string",
+      "field_type": "text"
+    },
+    {
+      "name": "MariaDB Password",
+      "description": "MariaDB password",
+      "env_variable": "MARIADB_PASS",
+      "default_value": "",
+      "user_viewable": true,
+      "user_editable": true,
+      "rules": "required|string",
+      "field_type": "text"
+    },
+    {
+      "name": "MariaDB Database",
+      "description": "MariaDB schema name",
+      "env_variable": "MARIADB_DB",
+      "default_value": "voicetwine",
```

Comment:
This can not be done as all our databases start with `s<number>_` so at least put a value that can be a good default